init/Bootstrap.js

```diff
@@ -7,17 +7,19 @@
 
 import * as Webix from "../js/webix/webix.js";
 // NOTE: changed to require() so switching to webix_debug.js will work.
-// var Webix = require("../js/webix/webix-debug.js");
+// const Webix = require("../js/webix/webix-debug.js");
 
 import webixCSS from "../js/webix/webix.css";
 // Make sure webix is global object
 if (!window.webix) {
    window.webix = Webix;
 }
 
-import "../js/webix/locales/th-TH.js";
+import("../js/webix/locales/th-TH.js");
```

Comment:
I'm not sure that using a dynamic import is best here. As a dynamic import webpack will save it as separate file that loads form the server. This is an extra request that happens each load.

We could do this import later when we check the user locale and only load it if we need to But I think it's simpler to just do a standard import here.